File: paiza/ruby/C063.rb
# 自分の得意な言語で
# Let's チャレンジ！！

N = gets.chomp.to_i
flowering_day = []
flowers = [0]

N.times do |i|
    ipt = gets.chomp.split(' ').map!(&:to_i)
    flowering_day << ipt.sum
end


1.upto(flowering_day.max) do |j|
    flowers[j] = flowering_day.count(j)
end


flowers.length.times do |k|
    if flowers[k] == flowers.max
        p k
        break
    end
end
    

File: paiza/python/C/C063.py
# coding: utf-8
# 自分の得意な言語で
# Let's チャレンジ！！

N = int(input())

FloweringDay = []

for i in range(N):
    FloweringDay.append(sum(list(map(int, input().split()))))

max_n = 0

for day in list(set(FloweringDay)):
    flower_n = FloweringDay.count(day)
    if flower_n > max_n:
        max_n = flower_n
        max_day = day

print(max_day)
